listenbrainz/webserver/views/playlist_api.py

```diff
@@ -860,22 +860,33 @@ def export_playlist(playlist_mbid, service):
     if not is_valid_uuid(playlist_mbid):
         log_raise_400("Provided playlist ID is invalid.")
 
-    if service != "spotify":
+    if service != "spotify" and service != "soundcloud" and service != "apple_music":
```

Comment:
Just checking here: should this change also include soundcloud?
Might be missing a `elif service == "soundcloud":` around line 890 to support it?